tools/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ if (NOT ${SPIRV_SKIP_EXECUTABLES})
  add_subdirectory(lesspipe)
 endif()
 add_subdirectory(emacs)
+add_subdirectory(dump)
```

Comment:
You might need to update the build.gn and android.mk.

Reply:
I thought of that, but didn't wanted to add it to those 2 directories for 1 reason:
 - I cannot test those easily, hence validate it works.
 Reasoning was: if somebody on chrome/android needs it, they can add it and test.
 But I can understand we want to be cross-platform too.